Server/emailHelper-dist.sh

```diff
@@ -29,6 +29,12 @@ smtpServer=""
 smtpAuth=""
 smtpPass=""
 
+if [[ ${IN_DOCKER} && ${SMTP_SERVER} && ${SMTP_AUTH} && ${SMTP_PASS} ]]; then
+	smtpServer=$SMTP_SERVER
```

Comment:
Walking through the docker setup in the readme, I did not see where to provide these.

Reply:
assume this is one of the comments you figured out later - in the docker/README.md (https://github.com/logicnow/BlueSky/pull/14/files#diff-a489144702a723431811d56a3c44690a) it lists these 3 SMTP ENV variables :)